Dockerfile

```diff
@@ -37,5 +33,4 @@ ENV PS4=+
 # Travis stuff
 ARG TRAVIS_JOB_ID
 ENV TRAVIS_JOB_ID=${TRAVIS_JOB_ID}
```

Comment:
also in `.travis.yml`

`docker build -t kcov_img . --build-arg TRAVIS_JOB_ID=$TRAVIS_JOB_ID` 

=>

`docker build -t kcov_img .`